hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -832,9 +824,6 @@ func reconcileHostedControlPlane(hcp *hyperv1.HostedControlPlane, hcluster *hype
 	if hcluster.Spec.Etcd.ManagementType == hyperv1.Managed && hcluster.Spec.Etcd.Managed != nil {
 		hcp.Spec.Etcd.Managed = hcluster.Spec.Etcd.Managed.DeepCopy()
 	}
-	if hcluster.Spec.ImageContentSources != nil {
```

Comment:
Why is this removed? Rebase issue or on purpose?

Reply:
rebase issue